testdata/baselines/reference/submodule/compiler/jsEnumCrossFileExport.symbols.diff

```diff
@@ -47,7 +55,9 @@
 ->Other.Cls : Symbol(Other.Cls, Decl(index.js, 0, 15))
 ->Other : Symbol(Other, Decl(index.js, 0, 3), Decl(index.js, 0, 15))
 ->Cls : Symbol(Other.Cls, Decl(index.js, 0, 15))
++>Other.Cls : Symbol(Cls, Decl(index.js, 0, 15))
```

Comment:
I feel like so many of the leftover diffs are just this symbol naming problem

Reply:
yepppp. I need to look at it to see if it's jsdoc's fault or our symbol display.